src/main/java/com/synopsys/integration/jira/common/cloud/rest/service/IssueSearchService.java

```diff
@@ -41,18 +41,25 @@ public IssueSearchService(final JiraCloudService jiraCloudService) {
         this.jiraCloudService = jiraCloudService;
     }
 
-    public IssueSearchResponseModel findIssuesByComment(final String comment) throws IntegrationException {
```

Comment:
We could leave this method as well just to allow further uses if we need it.

Reply:
We have the raw JQL method if we want to do things outside of these constraints.